vcd/resource_vcd_nsxt_edgegateway_test.go

```diff
@@ -106,6 +109,23 @@ func TestAccVcdNsxtEdgeGateway(t *testing.T) {
 	postTestChecks(t)
 }
 
+// in CDS cluster ID isn't accessible: Forbidden: User is not authorized to perform this operation on the application. Please contact the system administrator to get access., error code 401
+func ifPossibleAddClusterId(t *testing.T, err error, vcdClient *VCDClient, params StringMap) {
+	clusterId, err := lookupAvailableEdgeClusterId(t, vcdClient)
+	if err != nil {
+		t.Logf("\nWARNING: cluster id fetch failed, test will continue withouth cluster id. Error: %s", err)
+		params["EdgeClusterForAssert"] = regexp.MustCompile(`[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$`)
```

Comment:
What is the purpose of adding a regexp to a params item? The result of `regexp.MustCompile` is a `*regexp.Regexp`. An explanatory comment is needed here.

Reply:
added more comments, we use params not only in template, but in assertion too.